### DISCUSSION TITLE
How can I forbid to tab title rolling?

### DISCUSSION THREAD

**Discussion by luotom** [qutebrowser/qutebrowser#6809] [Q&A] [answered]
What configure I should add to forbid the tab title always rolling?

![doutub_gif](https://user-images.githubusercontent.com/43427583/140375361-ce19659d-1b21-43ca-9b0c-12bc5d7a4c68.gif)

**Comment by toofar** [comment, marked as answer]
I don't think that's a feature. Do you know what is happening to make it do that? Scrolling, something on the page animating? It starts happening after you resize the window?
Also what does :version say about your environment?
